Examples/Tests/ion_stopping/inputs_3d

```diff
@@ -0,0 +1,96 @@
+my_constants.max_ion_energy = 100.  # eV
+my_constants.max_ion_velocity12 = sqrt(2.*max_ion_energy*q_e/m_p)/clight
+my_constants.max_ion_velocity34 = sqrt(2.*max_ion_energy*q_e/(4.*m_p))/clight
+
+max_step = 10
+
+amr.n_cell = 8 8 8
+amr.max_level = 0
+geometry.dims = 3
+geometry.prob_lo = -1. -1. -1.
+geometry.prob_hi = +1. +1. +1.
+boundary.field_lo = periodic periodic periodic
+boundary.field_hi = periodic periodic periodic
+boundary.particle_lo = periodic periodic periodic
+boundary.particle_hi = periodic periodic periodic
+algo.particle_shape = 1
+
+particles.species_names = ions1 ions2 ions3 ions4
+
+ions1.species_type = proton
+ions1.injection_style = "MultipleParticles"
+ions1.multiple_particles_pos_x = 0. 0. 0. 0.
+ions1.multiple_particles_pos_y = 0. 0. 0. 0.
+ions1.multiple_particles_pos_z = 0. 0. 0. 0.
+ions1.multiple_particles_vel_x = max_ion_velocity12 0. 0. max_ion_velocity12/2.
+ions1.multiple_particles_vel_y = 0. max_ion_velocity12 0. max_ion_velocity12/2.
+ions1.multiple_particles_vel_z = 0. 0. -max_ion_velocity12 -max_ion_velocity12/2.
+ions1.multiple_particles_weight = 1. 1. 1. 1.
+
+ions2.species_type = proton
+ions2.injection_style = "MultipleParticles"
+ions2.multiple_particles_pos_x = -0.1 +0.1 -0.1 +0.1
+ions2.multiple_particles_pos_y = -0.1 -0.1 +0.1 +0.1
+ions2.multiple_particles_pos_z = 0. 0. 0. 0.
+ions2.multiple_particles_vel_x = max_ion_velocity12 max_ion_velocity12 max_ion_velocity12 max_ion_velocity12
+ions2.multiple_particles_vel_y = 0. 0. 0. 0.
+ions2.multiple_particles_vel_z = 0. 0. 0. 0.
+ions2.multiple_particles_weight = 1. 1. 1. 1.
+
+ions3.charge = q_e
+ions3.mass = 4*m_p
+ions3.injection_style = "MultipleParticles"
+ions3.multiple_particles_pos_x = 0. 0. 0. 0.
+ions3.multiple_particles_pos_y = 0. 0. 0. 0.
+ions3.multiple_particles_pos_z = 0. 0. 0. 0.
+ions3.multiple_particles_vel_x = max_ion_velocity34 0. 0. max_ion_velocity34/2.
+ions3.multiple_particles_vel_y = 0. max_ion_velocity34 0. max_ion_velocity34/2.
+ions3.multiple_particles_vel_z = 0. 0. -max_ion_velocity34 -max_ion_velocity34/2.
+ions3.multiple_particles_weight = 1. 1. 1. 1.
+
+ions4.charge = q_e
+ions4.mass = 4*m_p
+ions4.injection_style = "MultipleParticles"
+ions4.multiple_particles_pos_x = -0.1 +0.1 -0.1 +0.1
+ions4.multiple_particles_pos_y = -0.1 -0.1 +0.1 +0.1
+ions4.multiple_particles_pos_z = 0. 0. 0. 0.
+ions4.multiple_particles_vel_x = max_ion_velocity34 max_ion_velocity34 max_ion_velocity34 max_ion_velocity34
+ions4.multiple_particles_vel_y = 0. 0. 0. 0.
+ions4.multiple_particles_vel_z = 0. 0. 0. 0.
+ions4.multiple_particles_weight = 1. 1. 1. 1.
+
+collisions.collision_names = stopping_on_electrons_constant stopping_on_electrons_parsed stopping_on_ions_constant stopping_on_ions_parsed
+
+stopping_on_electrons_constant.type = background_stopping
+stopping_on_electrons_constant.species = ions1
+stopping_on_electrons_constant.background_type = electrons
+stopping_on_electrons_constant.background_mass = m_e
```

Comment:
Strictly no, since the background mass defaults to the electron density, but the code does allow the mass to be specified. Note that the physics in this case only assumes that the ion velocity << background particle thermal velocity and ion mass >> background mass. This can be used any place where those relations hold, so I left the mass as an input parameter.